src/main/java/org/testng/TestRunner.java

```diff
@@ -628,6 +629,7 @@ private void privateRunJUnit() {
     // may result in an class resolution exception under different JVMs
     // The resolution process is not specified in the JVM spec with a specific implementation,
     // so it can be eager => failure
+    ITestNGMethod[] m_allTestMethods = computeAndGetAllTestMethods();
```

Comment:
According to the current usage of `m_allTestMethods` here, I propose to drop it.

Reply:
Fixed